src/components/inputs/amount-range.svelte

```diff
@@ -15,6 +17,17 @@
   $: maxValueFormatted = max / 10 ** decimals;
   $: valid = value >= min && value <= max;
 
+  onMount(() => {
+    value = Math.max(min, Math.min(max, value));
+    valueFormatted = value / 10 ** decimals;
+  });
+
+  // Make sure the component updates the internal state when value prop changes
+  afterUpdate(() => {
```

Comment:
im not sure about this, you are blocking the possibility of editing the input manually without using the range